seqr/views/apis/awesomebar_api.py

```diff
@@ -14,17 +15,17 @@
 MAX_STRING_LENGTH = 100
 
 
-def _get_matching_objects(query, project_guids, object_cls, filter_fields, object_fields, get_title, get_href, get_description=None, project_field=None, select_related_project=True, exclude_criteria=None):
+def _get_matching_objects(query, project_guids, object_cls, core_fields, href_expression, description_content=None,
+                          project_field=None, select_related_project=True, exclude_criteria=None):
     """Returns objects that match the given query string, and that the user can view, for the given object criteria.
 
     Args:
         user: Django user
         query: String typed into the awesomebar
         object_cls: Django model class of the object
-        filter_fields: Array of field names to match the query against
-        get_title: Function to get the title from an object
-        get_href: Function to get the href from an object
-        get_description: Optional function to get the description from an object
+        core_fields: Array of field names to match the query against and display the title
+        href_expression: Function to get the href from the
+        description_content: Optional function to get the description from the db
         project_field: Optional string defining the relationship between the object and parent project
```

Comment:
I think the docstring is pretty out of date (because its so hard to maintain) and we don't really use them in seqr anymore anyways, so I'm just going to get rid of it